src/DataStructures/Tensor/Expressions/AddSubtract.hpp

```diff
@@ -82,30 +82,25 @@ struct AddSub<T1, T2, ArgsList1<Args1...>, ArgsList2<Args2...>, Sign>
 
   AddSub(T1 t1, T2 t2) : t1_(std::move(t1)), t2_(std::move(t2)) {}
 
-  template <typename... LhsIndices, typename T, int U = Sign,
-            Requires<U == 1> = nullptr>
+  template <typename... LhsIndices, typename T, int U = Sign>
```

Comment:
You no longer need the `U = Sign`, you can just use `Sign` directly below. The `U = Sign` was needed because `Reqires` only works on the template parameters of the function, but `if constexpr` doesn't care where the template parameter comes from.

Reply:
Done! Also changed the `else if` to `else if constexpr`, as based on some reading I was doing, it seemed like it was also necessary. Let me know if I have this incorrect.

Reply:
Yes, I believe that's correct. If I remember correctly, I think you can do `else {` and the else is `constexpr`. I'll leave it up to you whether you want the `else if constexpr` or just `else`